src/jmc/compile/command/builtin_function/load_only.py

```diff
@@ -269,7 +270,8 @@ def call(self) -> str:
 )
 class ItemCreateSign(JMCFunction):
     _VARIANTS = {"oak", "spruce", "birch", "jungle",
-                 "acacia", "dark_oak", "crimson", "warped"}
+                 "acacia", "dark_oak", "crimson", "warped", 
+                 "mangrove", "bamboo", "cherry"}
```

Comment:
![image](https://github.com/WingedSeal/jmc/assets/91001643/75a5172e-bb75-4ce4-95cb-475039a5e61d)
Remember for supporting older versions

Reply:
you are right
instead I should remind myself instead :p